### PR TITLE
Aaronmgdr/celofy

### PR DESCRIPTION
**Pull request by aaronmgdr**
## What it solves
Make is Celo Specific and Remove Gnosis Trademarks

### Branding
* find replace on most uses of gnosis with celo 
* replaces gnosis logo with celo safe logo
* change color scheme to use celo colors 
* use Celo Brand Font-faces

## docs 
* switch rinkeby to alfajores in docs and readmes

### Networks
* Remove unsupported networks
* add baklava
* update network config files to use corect rpc endpoints end block explorers 

# other 
* remove intercom
* remove help center

## Screenshots

Screen Shot 2022-02-17 at 2 09 04 PM

https://user-images.githubusercontent.com/3814795/154579260-8264346d-85eb-417c-9d6c-25c36a410b26.mov

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,26 @@
...
 
 ## Production deployments
 
 This repository contains the code for the frontend code hosted at https://gnosis-safe.io/app/
 
-Besides the Ethereum Mainnet, the following networks are supported:
+Besides the Celo Mainnet, the following networks are supported:
+
+- [Alfajores Testnet](https://rinkeby.gnosis-safe.io/app/)
+- [Baklava Testnet](https://rinkeby.gnosis-safe.io/app/)
```
Comment (aaronmgdr) [inline README.md:22]: TODO get Correct URLS here

File: src/components/Root/index.tsx [hunk 2]
```diff
@@ -15,8 +15,26 @@ import './index.module.scss'
 import './OnboardCustom.module.scss'
 import './KeystoneCustom.module.scss'
 
+type StyledThemeType = typeof styledTheme
+
+const ourStyledTheme: StyledThemeType = {
```
Comment (aaronmgdr) [inline src/components/Root/index.tsx:20]: turns out safe-react uses both material ui components and styled components. so there are two theme providers

File: src/config/index.ts [hunk 3]
```diff
@@ -39,9 +39,7 @@ export const getNetworkLabel = (id: ETHEREUM_NETWORK): string => {
   return cfg ? cfg.network.label : ''
 }
 
-export const usesInfuraRPC = [ETHEREUM_NETWORK.MAINNET, ETHEREUM_NETWORK.RINKEBY, ETHEREUM_NETWORK.POLYGON].includes(
-  getNetworkId(),
-)
+export const usesInfuraRPC = false
```
Comment (aaronmgdr) [inline src/config/index.ts:42]: will need to use forno right? so never infura rpc?

File: src/config/networks/alfajores.ts [hunk 4]
```diff
@@ -6,12 +6,11 @@ import {
   NetworkConfig,
   WALLETS,
 } from 'src/config/networks/network.d'
-import { ETHGASSTATION_API_KEY } from 'src/utils/constants'
 
 const baseConfig: EnvironmentSettings = {
```
Comment (aaronmgdr) [inline src/config/networks/alfajores.ts:10]: some of the urls here are certainly wrong. some i think are correct some i know are

File: src/config/networks/mainnet.ts [hunk 5]
```diff
@@ -18,8 +17,8 @@ const baseConfig: EnvironmentSettings = {
       gweiFactor: '1e8',
     },
   ],
-  safeAppsRpcServiceUrl: 'https://mainnet.infura.io:443/v3',
-  rpcServiceUrl: 'https://mainnet.infura.io:443/v3',
+  safeAppsRpcServiceUrl: 'https://forno.celo.org',
+  rpcServiceUrl: 'https://forno.celo.org',
```
Comment (aaronmgdr) [inline src/config/networks/mainnet.ts:21]: This seems nessesessary but also if it was working before maybe not?

File: src/routes/load/Load.test.tsx [hunk 6]
```diff
@@ -132,7 +132,7 @@ describe('<Load>', () => {
       })
     })
 
-    it('Should get Safe Address From ENS Name Domain', async () => {
+    it.skip('Should get Safe Address From ENS Name Domain', async () => {
```
Comment (aaronmgdr) [inline src/routes/load/Load.test.tsx:135]: I assume this isnt valid for us as ENS doesnt exist on celo?

File: src/logic/collectibles/utils/index.ts [hunk 7]
```diff
@@ -21,13 +9,7 @@ export const SAFE_TRANSFER_FROM_WITHOUT_DATA_HASH = '42842e0e'
...
  * @returns string
  */
-export const getTransferMethodByContractAddress = (contractAddress: string): string => {
-  if (sameAddress(contractAddress, CK_ADDRESS[getNetworkId()])) {
-    // on mainnet `transferFrom` seems to work fine but we can assure that `transfer` will work on both networks
-    // so that's the reason why we're falling back to `transfer` for CryptoKitties
-    return 'transfer'
-  }
-
+export const getTransferMethodByContractAddress = (): string => {
```
Comment (aaronmgdr) [inline src/logic/collectibles/utils/index.ts:12]: I remove this as its a hack for Crypto kitties contract which isnt relevent on celo

**Review by nicolasbrugneaux — APPROVED** [review]

**Pull request merged**